modules/cnv-labels-nodes-workloadscheduling.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv_advanced_vm_management/cnv-dedicated-resources-vm.adoc
+// * cnv/cnv_vm_templates/cnv-dedicated-resources-vm-template.adoc
+
+[id="cnv-labels-nodes-workloadscheduling_{context}"]
+
+= Preparing to enable dedicated resources
```

Comment:
Recommend changing the title to 'Preparing nodes to enable dedicated resource'
Also change the id and file name to reflect the change

Reply:
Changed ID and file name with git mv as requested.